devtools/packaging/scripts/osx/build.sh

```diff
@@ -37,7 +38,7 @@ fi
 mkdir build
 cd build
 cmake ../openmm $CMAKE_FLAGS
-make -j4 all DoxygenApiDocs sphinxpdf
+make -j4 all PythonInstall C++ApiDocs PythonApiDocs sphinxpdf
```

Comment:
Yes, but I keep asking: how did this work before?  *None of this behavior has changed*.  You always had to `make install` before `make PythonInstall`, and you always had to build the Python API before you could build the documentation for it.  What has changed?